weekly-standup-template.md

```diff
@@ -121,16 +117,20 @@ Please include links.
 <!--
 ##### Your name here
 
```

Comment:
Will focus on xx and yy, for which I will zz.

Reply:
I think that is right. We want to be fairly specific here so that people know what everyone is doing.

Saying "Story X" isn't so useful.
Fix in a12629a